data/story oathkeepers.txt

```diff
@@ -0,0 +1,168 @@
+			choice
+				`	"Yeah, load it up and I'll go."`
+				accept
+				`	"Soon, I've got some shopping to do first."`
+					defer
+	on complete
+		dialog `As you move to land on Ada to install the sensors, you find that several ships with Northern Union colors are already landed, and they're likely stealing from the weapons developers. It seems to be a relatively small raid... with some skill, you might be able to destroy it alone.`
+		
+##Not so sure about these missions working, the on offer for 1.1 is meant to show when you take off. Also, all fleets in this .txt are placeholder.
+mission "Oathkeeper Scouting 1.1"
+	name `Retreat from Ada`
+	description `Escape the Northern Union fleet that followed you off of Ada, and retreat to Farpoint.`
+	to offer
+		has "Oathkeeper Scouting 1: complete"
+	to fail
+		has "Oathkeeper Scouting 1.2: complete"
+	source "Ada"
+	destination "Farpoint"
+	on offer
+		dialog `Looks like some of the ships have followed you into orbit! You can fight them, or run.`
+	on complete
+		dialog `You arrive on Farpoint and contact Mulgrew, who seems disappointed that the Northern Union likely got away with the raid, but glad that you made it out okay. She thanks you and tells you to meet her in the spaceport later.`
+		"ran" ++
+		
+mission "Oathkeeper Scouting 1.2"
+	name `Defend Ada`
+	description `Destroy the raiding fleet attacking Ada and land on the planet to install the sensors for Mulgrew.`
+	to offer
+		has "Oathkeeper Scouting 1: complete"
+	to fail
+		has "Oathkeeper Scouting 1.1: complete"
+	source Ada
+	destination Ada
+	
+	npc kill
+		personality heroic staying
+		government "Northern Union"
+		system Aldebaran
+		fleet "Small Northern Union" 2
+	
+	on complete
+		dialog `You land on the defended Ada and install the sensors, then contact Mulgrew and explain what happened. She seems very impressed that you took down the entire raiding fleet, and she promises to send you a bonus of <payment> and talk to you at Farpoint about your next job.`
+		payment 30000
+		fought ++
+		
+mission "Oathkeeper Scouting 2"
+	name `Scout Elnath`
+	description `After placing the sensors on Ada, Mulgrew has asked you to scout the Elnath system for any other raiders.`
+	to offer
+		or
+			has "Oathkeeper Scouting 1.1: complete"
+			has "Oathkeeper Scouting 1.2: complete"
+	source "Farpoint"
+	destination "Pilot"
+	
+	on offer
+		conversation
+			`You meet with Mulgrew again, and she seems a bit more stressed than usual, but she still has plenty to say. "Hey, <first>! I'm sure I already said this, but I'm glad you made it out of that mess on Ada alright. Good thing too, because you'll be doing lots of that work in the foreseeable future!" Mulgrew chuckles.

```

Comment:
```suggestion
			`You meet with Mulgrew again, and she seems a bit more stressed than usual, but she still has plenty to say. "Hey, <first>! I'm sure I already said this, but I'm glad you made it out of that mess on Ada alright. Good thing too, because you'll be doing lots of that work in the foreseeable future!" Mulgrew chuckles.`

```